CHANGES.txt

```diff
@@ -35,6 +39,9 @@ Backward Incompatibilities
   depending on it directly within your project.
   See https://github.com/Pylons/pyramid/pull/3140
 
+- On CherryPy 9+ all functionality from ``cherrypy.wsgiserver`` was moved to ``cheroot``.
+  See https://github.com/Pylons/pyramid/issues/3195
```

Comment:
As I noted, I think we can drop this mention:  users won't need to adjust their code, which means this is not a backward-incompatibility *for Pyramid.*